pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SchemasResourceBase.java

```diff
@@ -294,4 +302,6 @@ private void validateDestinationAndAdminOperation(boolean authoritative) {
             }
         }
     }
+
+    private static final Logger log = LoggerFactory.getLogger(SchemasResourceBase.class);
```

Comment:
nit:
please use `@Slf4j` annotation from lombok

Reply:
Thanks. I just copied this usage from other Class like 'TenantsBase' in same package, we could change to `@Slf4j` for all those Class in another PR.